contrib/ansible/deploy-playbook.yml

```diff
@@ -34,13 +34,21 @@
     cluster_api_image: "registry.svc.ci.openshift.org/openshift-cluster-operator/kubernetes-cluster-api:latest"
     cluster_api_image_pull_policy: "Always"
 
+    use_real_images: False
+    machine_controller_imagestream: "fake-machine-controller"
```

Comment:
I'm inclined to think that if you're calling deploy-playbook, use_real_images should default to true. That would only be done for real world deployments, whereas it would make sense to default to false for deploy-devel. Setting it when calling devel should override the default here.

Reply:
Could you also add a glaring entry in the README outlining the default behavior and how to work around it.